### PR TITLE
Start nuklear porting

### PR DESCRIPTION
**Pull request by AleksandraButrova**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,3 +35,4 @@ third-party/qt/.downloaded
 third-party/qt/qt-everywhere-opensource-src-4.8.4.tar.gz
 third-party/ti/ipc_1_24_00_16
 third-party/ti/syslink_2_21_01_05
+/.vscode
```
Comment (anton-bondarev) [inline .gitignore]: Please remove this duplicated line, and don't forget newline.

File: src/cmds/grfxs/grfx.c [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+	/* end of work */
+    nk_font_atlas_clear(&atlas);
+    nk_free(&ctx);
+    
+	/* checking working command */
+	printf("\nJob's done! :)\n\n");
+
+	return 0;
+}
```
Comment (anton-bondarev) [inline src/cmds/grfxs/grfx.c]: Please add a new line

File: src/cmds/grfxs/grfx_canvas.c [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+    nk_buffer_free(&device.cmds);
+
+
+
+
+
+    printf("\nEnd of program.\nIf you see it then something goes wrong.\n");
+    return 0;
+
+}
```
Comment (anton-bondarev) [inline src/cmds/grfxs/grfx_canvas.c]: a new line

Comment (AleksandraButrova) [inline src/cmds/grfxs/grfx_canvas.c]: Where?

File: src/cmds/grfxs/grfx.c (hunk 2 above)
Comment (AleksandraButrova) [inline src/cmds/grfxs/grfx.c]: Where?

**Comment by anton-bondarev** [comment]
Closed by #1127

**Pull request closed without merging**